src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs

```diff
@@ -36,16 +37,20 @@ public void ConfigureServices(IServiceCollection services)
             var clients = _configuration.GetSection("clients").Get<ClientSettingCollection>();
             var resources = _configuration.GetSection("resources").Get<ApiResourceSettingCollection>();
             var identityResources = _configuration.GetSection("identityResources").Get<IdentityResourceSettingCollection>();
+            var certificateSettings = _configuration.GetSection("certificateSettings").Get<CertificateSettings>();
 
             var issuerUrl = _configuration.GetValue<string>("issuerUrl");
 
             Log.Logger.Information("Clients: {@clients}", clients);
             Log.Logger.Information("Api Resources: {@resources}", resources);
             Log.Logger.Information("Identity Resources: {@identityResources}", identityResources);
             Log.Logger.Information("Issuer Url on IdentityAPI is: {@issuerUrl}", issuerUrl);
+            Log.Logger.Information(
+                "Certificate Settings on IdentityAPI is: {settings}", certificateSettings);
```

Comment:
Can this be kept to one line? (It should still be shorter than some of the lines above it.)

Although if you think this looks better leave it as is.